### DISCUSSION TITLE
How refresh data only if the page is out of focus?

### DISCUSSION THREAD

**Discussion by fivetran-dmitrysemenov** [TanStack/query#2530] [Q&A]
I have a task to refresh data only if the page is out of focus. I used the params below, but it didn't help me. 
```js
{
    refetchInterval: 3000,
    refetchIntervalInBackground: true,
    refetchOnWindowFocus: false
}
```

**Comment by TkDodo** [comment]
the above config should trigger a request every 3 seconds if the page is active and if the page is in the background, so basically, every 3 seconds no matter where you are. What is not working with the config, and can you reproduce it in a codesandbox?
